TimeOffTracker.WebApi/Controllers/AccountController.cs

```diff
@@ -42,7 +48,11 @@ public async Task<HttpStatusCode> Post([FromForm] RegisterViewModel model)
         {
             if (ModelState.IsValid)
             {
-                User user = new User { Email = model.Email, UserName = model.Email };
+                User user = new User 
```

Comment:
Use Mapper.Map<RegisterViewModel, User>() instead of manual mapping

Reply:
For this, I transferred the RegisterViewModel from WEB to ApiModels project.